content/en/docs/reference/command-line-tools-reference/feature-gates/windows-host-process-containers.md

```diff
@@ -0,0 +1,5 @@
+---
+title: WindowsHostProcessContainers
+content_type: feature_gate
```

Comment:
Why does this page not contain the `_build` object?
@sftim , can you explain what the `_build` object does when the content from the page is rendered?

Reply:
Could use a template page for the process, if we expect authors to create one of these pages when they add new feature gate content.

Reply:
`_build` controls page rendering at static site build time. Maybe I missed it out by accident.

We could add a template page and even a linter.